src/platform/silabs/efr32/wifi/wfx_host_events.h

```diff
@@ -141,14 +139,19 @@ typedef struct __attribute__((__packed__)) sl_wfx_mib_req_s
 #define WLAN_TASK_STACK_SIZE 1024
 #define WLAN_TASK_PRIORITY 1
 #define WLAN_DRIVER_TASK_PRIORITY 1
+#define BLE_DRIVER_TASK_PRIORITY 1
```

Comment:
This means all these tasks are round-robbin ? Is that OK?

It also means all these tasks are very low priority. Did you intend that?

Reply:
I will open up an internal ticket to get these fixed and tested over a full QA cycle.